lib/std/os/linux/io_uring.zig

```diff
@@ -1260,6 +1260,36 @@ pub const IO_Uring = struct {
         }
     }
 
+    pub const IO_WQ_WorkerLimit = struct {
+        max_bounded_workers: u32,
+        max_unbounded_workers: u32,
+    };
+
+    // Sets an upper bound for workers used to perform work required for bounded work (e.g. regular file or block device
+    // operations) and unbounded work (i.e. requests that may never complete e.g. network requests)
+    // Passing 0 for one of the limits will leave the current limit unchanged.
+    // On success, returns the previously set limits
+    // Available since 5.15
+    // Note: These pools of workers are shared only among io_uring instances that have been created by the same thread,
+    //       in the same process and consequently the limits set with this function only affect io_uring instances at
+    //       this scope.
```

Comment:
If you make these triple slashes, they become doc comments and then show up in the auto-generated documentation for this function.